deprecated/ActionScheduler_WPCLI_Scheduler_command.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/**
+ * Class ActionScheduler_WPCLI_Scheduler_command
+ *
+ * Retain public API for backwards compatibility.
+ */
+class ActionScheduler_WPCLI_Scheduler_command {
```

Comment:
The `c` in `Command` should be upper case.

Reply:
@rrennick This change is from #267 (please see https://github.com/Prospress/action-scheduler/pull/267#issuecomment-477174034).

Reply:
@rrennick The class name in `master` has a lower `c`... this class is being added for backwards compatibility (#272), so matching the class name is needed.

Reply:
Apparently, class names are not case-sensitive. 🤔

Reply:
Capitalized "C" in https://github.com/Prospress/action-scheduler/pull/280/commits/af18ec5a465cebcd3cf4c4028e24e8c93680443b.